presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -269,4 +271,16 @@ public FeaturesConfig setRegexLibrary(RegexLibrary regexLibrary)
         this.regexLibrary = regexLibrary;
         return this;
     }
+
+    public boolean isCharPadSpaces()
+    {
+        return charPadSpaces;
+    }
+
+    @Config("char.pad-spaces")
```

Comment:
I'm not very familiar with SQL collation rules, but looking through the spec they look very complicated. Can you open an issue, so that we can discuss how collation should work in Presto, and start things off with your understanding of the spec?